datashader/glyphs/line.py

```diff
@@ -17,23 +19,41 @@
     cuda_args = None
 
 
+# Eventually this will be replaced with attributes and/or
```

Comment:
Making promises in comments isn't usually a good idea, unless it's a promise to be fulfilled in this PR. Otherwise those stay around in the code confusing people! Here e.g. it could say "It would be cleaner to replace these constants with ...".

Reply:
Corrected.